packages/frontend-core/src/api/app.js

```diff
@@ -123,6 +123,15 @@ export const buildAppEndpoints = API => ({
     })
   },
 
+  /**
+   * Gets budibase platform debug information.
+   */
+  fetchSystemDebugInfo: async () => {
+    return await API.get({
+      url: `/api/dev/debug`,
```

Comment:
I would vote for `/api/diagnostics`, given this isn't specifically related to `dev`